apps/builder/app/shared/shim.ts

```diff
@@ -0,0 +1,17 @@
+export const setDifference = <Item>(current: Set<Item>, other: Set<Item>) => {
```

Comment:
setDifference reads like set method like setX etc ... I was starring at it for a while until I realized you mean a difference for Set

Reply:
yeah, naming is hard